packages/amplication-build-manager/src/build-runner/build-runner.controller.spec.ts

```diff
@@ -105,7 +106,7 @@ describe("BuildRunnerController", () => {
     ).resolves.not.toThrow();
 
     expect(mockKafkaServiceEmitMessage).toBeCalledWith(
-      configService.get(Env.CODE_GENERATION_SUCCESS_TOPIC),
+      configService.get(KAFKA_TOPICS.CODE_GENERATION_SUCCESS_TOPIC),
```

Comment:
Same as here https://github.com/amplication/amplication/pull/6698/files#diff-243e5dc75c8a6ca029dac34dadf96afd121d56b8c6895888240190d8a1a3f4d2R24 
```suggestion
      KAFKA_TOPICS.CODE_GENERATION_SUCCESS_TOPIC,
```